apps/ledger-live-mobile/src/screens/SendFunds/SummaryNft.tsx

```diff
@@ -20,21 +17,36 @@ type Props = {
 const SummaryNft = ({ transaction, currencyId }: Props) => {
   const { colors } = useTheme();
   const { t } = useTranslation();
-  const tokenId = (transaction as EthereumTransaction)?.tokenIds?.[0];
-  const quantity = (transaction as EthereumTransaction)?.quantities?.[0];
-  const data = useNftMetadata(
-    (transaction as EthereumTransaction)?.collection,
-    tokenId,
-    currencyId,
-  );
-  const metadata = (data as NFTResourceLoaded)?.metadata;
+  const tokenId = useMemo(() => {
+    if (transaction?.family === "evm") {
+      return transaction.nft.tokenId;
+    } else if (transaction?.family === "ethereum") {
+      return transaction.tokenIds?.[0];
+    }
+  }, [transaction]);
+  const quantity = useMemo(() => {
+    if (transaction?.family === "evm") {
+      return transaction.nft.quantity;
+    } else if (transaction?.family === "ethereum") {
+      return transaction.quantities?.[0];
+    }
+  }, [transaction]);
+  const collection = useMemo(() => {
+    if (transaction?.family === "evm") {
+      return transaction.nft.contract;
+    } else if (transaction.family === "ethereum") {
```

Comment:
We check if transaction is not null (?), why not here ?

If it's the better pattern, add ? everywhere, else remove them

Can transaction actually be null/undefined ?

Reply:
Good question I don't know why the linter would not tell me to use optional chaining here 🤔 
`family` isn't part of `TransactionCommon` so it shouldn't be happy about it.